src/hashmap.c

```diff
@@ -1,51 +1,36 @@
-#define BUCKETS 1000000
-#define PRIME_SEED_1 17
-#define PRIME_SEED_2 97
-#define KEY_CHARS 128
-#define VALUE_CHARS 128
-#define NODE_SIZE sizeof(Node)
```

Comment:
you can just use sizeof(Node) directly, no need to #define it. sizeof(Node) is a "const expression" meaning it is always known at compile time so you can use it to initialise stack arrays and such